src/index.html

```diff
@@ -7,6 +7,9 @@
 </head>
 <body>
   <div id="root"></div>
+
+  <a href="./about.html"></a>
```

Comment:
Add some text to this link to view it on the page

Reply:
About this, we don't really want this link to be seen, it is just here for parcle to pull in the file on serve/build time

Reply:
Got it